cmd/podman/kill.go

```diff
@@ -51,54 +50,45 @@ func init() {
 
 // killCmd kills one or more containers with a signal
 func killCmd(c *cliconfig.KillValues) error {
-	var (
-		killFuncs  []shared.ParallelWorkerInput
-		killSignal uint = uint(syscall.SIGTERM)
-	)
-
-	rootless.SetSkipStorageSetup(true)
-	runtime, err := libpodruntime.GetRuntime(&c.PodmanCommand)
-	if err != nil {
-		return errors.Wrapf(err, "could not get runtime")
+	if c.Bool("trace") {
+		span, _ := opentracing.StartSpanFromContext(Ctx, "killCmd")
+		defer span.Finish()
 	}
-	defer runtime.Shutdown(false)
 
-	if c.Signal != "" {
+	if c.Flags().Changed("signal") {
 		// Check if the signalString provided by the user is valid
 		// Invalid signals will return err
-		sysSignal, err := signal.ParseSignal(c.Signal)
-		if err != nil {
+		if _, err := signal.ParseSignal(c.Signal); err != nil {
```

Comment:
Why not do this always.  c.Signal defaults to KILL, so it should work nad no need to check the Changed flag above.

Reply:
done